transcendence-app/src/chat/chat-room-message-with-user.domain.ts

```diff
@@ -0,0 +1,18 @@
+import { User } from '../user/user.domain';
+import { ChatRoomMessageWithUserEntity } from './infrastructure/db/chat-room-message-with-user.entity';
+
+export class ChatRoomMessageWithUser {
```

Comment:
Don't see a real value of having this. The idea of the domain is to validate the minimal pieces of our problem

We already have two separate domains for User and Chat. We may do this joined models in upper layers

Reply:
On the `/chatroom` page, we will display the messages from a chat room, and each message contains information about the user (username and avatar)

How do you suggest we do these joined models in the upper layers?

Reply:
I merge this as is to work on the other tasks that require this table. If there is a better way, we can change it later.